### ISSUE TITLE
Style Issue: Network Traffic Display Wrapping on Server Probe Page

### ISSUE THREAD

**Issue by danblaze13** [msgbyte/tianji#20]
I have encountered a visual styling issue on the Server Probe page that affects the readability and aesthetics of network traffic data. When the network traffic figures are too long, possibly due to a high number of digits, the text wraps onto the next line in an unappealing way. This not only disrupts the layout but also makes it difficult to quickly assess the traffic information at a glance.

---

我在服务器探针页面遇到了一个视觉样式问题，该问题影响了网络流量数据的可读性和美观性。当网络流量数字太长时（可能是因为位数过多），文本会以一种不美观的方式换行到下一行。这不仅打乱了布局，而且还使得用户难以快速一目了然地评估流量信息。

<img width="1269" alt="iShot_2024-01-25_23 45 22" src="https://github.com/msgbyte/tianji/assets/70431454/97c082cc-d1af-48ae-926f-12e08cff781d">
<img width="1356" alt="iShot_2024-01-25_23 43 15" src="https://github.com/msgbyte/tianji/assets/70431454/e6d6141a-ea1e-4f30-b6a8-9a210667cfc1">

**Comment by moonrailgun** [comment]
did you mind check your tianji version. i am try to reproduce your issue but not going very well.

![image](https://github.com/msgbyte/tianji/assets/6964737/bf03089c-736d-47a4-8e5b-063739a3740a)

if you are using `v1.4.3`. font size should be smaller.

**Comment by danblaze13** [comment]
Yes, I'm using version 1.4.3, my device is a MAC, my system version is 14.2.1, and I'm using Edge as the browser, version 120.0.2210.144

I'll continue to monitor this issue and add any further information that may help in reproducing it!

**Comment by moonrailgun** [comment]
I dont know but simply add more space, its will be apply in next release version
